### PR TITLE
refactor: add tags to old calendar widget

### PR DESCRIPTION
**Pull request by davidma415**
Adding tags (similar to embed calendar widget)

### CODE REVIEW

**Review by jaredrooprai — APPROVED** [review]

File: plugin/showpass-wordpress-plugin-shortcode.php [hunk 1]
```diff
@@ -937,6 +937,9 @@ function wpshp_calendar_widget($atts, $content = null) {
 
   $organization_id = get_option('option_organization_id');
 
+  $tags = isset($atts['tags']) ? $atts['tags']
+  : null;
```
Comment (april-duong) [inline plugin/showpass-wordpress-plugin-shortcode.php:941]: minor: indentation

File: plugin/js/showpass-custom.js [hunk 2]
```diff
@@ -94,7 +94,8 @@
 			let id = $(this).attr('data-org-id');
 			let params = {
 				'theme-primary': $(this).attr('data-color') || $('#option_widget_color').val(),
-				'keep-shopping': false
+				'keep-shopping': false,
+                'tags': $(this).attr('data-tags')
```
Comment (april-duong) [inline plugin/js/showpass-custom.js:98]: not sure if it's github doing this but the indent is misaligned

**Review by april-duong — CHANGES_REQUESTED** [review]
Looks good, just minor indent issues

**Comment by davidma415** [comment]
@april-duong  weird indentations, but fixed them. should be good now.

**Review by april-duong — APPROVED** [review]

**Pull request merged**